modules/geo-layers/package.json

```diff
@@ -32,10 +32,16 @@
   "dependencies": {
     "h3-js": "^3.4.3",
     "long": "^3.2.0",
-    "s2-geometry": "^1.2.10"
+    "s2-geometry": "^1.2.10",
+    "@math.gl/geospatial": "^3.0.0-beta.3",
+    "@math.gl/culling": "^3.0.0-beta.3",
+    "@loaders.gl/3d-tiles": "^1.2.0"
   },
   "peerDependencies": {
     "@deck.gl/core": "^7.0.0",
-    "@deck.gl/layers": "^7.0.0"
+    "@deck.gl/layers": "^7.0.0",
+    "@deck.gl/mesh-layers": "^7.0.0",
+    "@loaders.gl/core": "^1.2.0-beta.4",
```

Comment:
- We should use the same version for all loaders.gl modules 1.2.2 is the latest will publish `1.3.0-beta.1` this week.
- Worth discussing if `core` and `gltf` should dependencies or peerDependencies. Since `3d-tiles` is a dependency, perhaps they might as well all be dependencies.